test/templates/dialog/component.njk

```diff
@@ -0,0 +1,35 @@
+{% import "../forms.njk" as forms %}
+{% from "../button/component.njk" import button %}
+{% from "../icon/component.njk" import icon %}
+<!--
+dialog:
+    properties={},
+-->
+{% macro dialog(properties={}) %}
+<div class="fd-dialog--{{properties.size}}" role="dialog">
```

Comment:
The size here is a modifier with can be passed as a modifier not a property. There is a modifier filter defined to handle the additional class. 

```
<div class="fd-dialog{{ modifier.block | modifier('dialog') }}" role="dialog">
```

See other components and how they handle modifiers, states and aria.